src/main/java/org/htwkvisu/gui/MapCanvas.java

```diff
@@ -86,14 +84,40 @@ public MapCanvas(ScoringCalculator calculator) {
      * @return BoundingBox Coordinates Boundingbox
      */
     public BoundingBox getCoordsBounds() {
-        return coordsBounds;
+        return this.coordsBounds;
```

Comment:
Habe gestern mit meinen Softwarearchitekten darüber geredet. This sollte benutzt werden wenn eine Variable mit gleichen Namen vorhanden ist, ansonsten stört es beim lese-Fluss, zumal getter auto-generiert werden und damit stimmen sollten.